commands/operator-sdk/cmd/scorecard/basic_tests.go

```diff
@@ -0,0 +1,206 @@
+// status blocks exist or return an error after the timeout.
+func checkSpecAndStat(runtimeClient client.Client, obj unstructured.Unstructured, noStore bool) error {
+	testSpec := scorecardTest{testType: basicOperator, name: "Spec Block Exists", maximumPoints: 1}
+	testStat := scorecardTest{testType: basicOperator, name: "Status Block Exist", maximumPoints: 1}
+	err := wait.Poll(time.Second*1, time.Second*time.Duration(SCConf.InitTimeout), func() (bool, error) {
+		err := runtimeClient.Get(context.TODO(), types.NamespacedName{Namespace: SCConf.Namespace, Name: name}, &obj)
+		if err != nil {
+			return false, fmt.Errorf("error getting custom resource: %v", err)
+		}
+		var specPass, statusPass bool
+		if obj.Object["spec"] != nil {
+			testSpec.earnedPoints = 1
+			specPass = true
+		}
+
+		if obj.Object["status"] != nil {
+			testStat.earnedPoints = 1
+			statusPass = true
+		}
+		return statusPass && specPass, nil
+	})
+	if !noStore {
+		scTests = append(scTests, testSpec, testStat)
+	}
+	if err != nil && !reflect.DeepEqual(err, wait.ErrWaitTimeout) {
+		return err
+	}
+	return nil
+}
+
+// TODO: user specified tests for operators
+
+// checkStatusUpdate looks at all fields in the spec section of a custom resource and attempts to modify them and
+// see if the status changes as a result. This is a bit prone to breakage as this is a black box test and we don't
+// know much about how the operators we are testing actually work and may pass an invalid value. In the future, we
+// should use user-specified tests
+func checkStatusUpdate(runtimeClient client.Client, obj unstructured.Unstructured) error {
+	test := scorecardTest{testType: basicOperator, name: "Operator actions are reflected in status", maximumPoints: 1}
+	err := runtimeClient.Get(context.TODO(), types.NamespacedName{Namespace: SCConf.Namespace, Name: name}, &obj)
+	if err != nil {
+		return fmt.Errorf("error getting custom resource: %v", err)
+	}
+	if obj.Object["status"] == nil || obj.Object["spec"] == nil {
+		scTests = append(scTests, test)
+		return nil
+	}
+	statCopy := make(map[string]interface{})
+	for k, v := range obj.Object["status"].(map[string]interface{}) {
+		statCopy[k] = v
+	}
+	specMap := obj.Object["spec"].(map[string]interface{})
+	err = modifySpecAndCheck(specMap, obj)
+	if err != nil {
+		test.earnedPoints = 0
+		scTests = append(scTests, test)
+		return nil
+	}
+	test.earnedPoints = 1
+	scTests = append(scTests, test)
+	return nil
+}
+
+// modifySpecAndCheck is a helper function for checkStatusUpdate
+func modifySpecAndCheck(specMap map[string]interface{}, obj unstructured.Unstructured) error {
+	statCopy := make(map[string]interface{})
+	for k, v := range obj.Object["status"].(map[string]interface{}) {
+		statCopy[k] = v
+	}
+	var err error
+	for k, v := range specMap {
+		mapType := false
+		switch t := v.(type) {
+		case int64:
+			specMap[k] = specMap[k].(int64) + 1
```

Comment:
This might be fine for now, but we should potentially use the fuzzer for the this.